.github/ISSUE_TEMPLATE/vsp-release-plan.md

```diff
@@ -0,0 +1,86 @@
+# Release Plan Template [Draft]
+This Release Plan Template is intended to be used to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necesssary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs. 
+
+You do not need to complete a release plan for a product or feature update that does not:
+
+- Add substantial new functionality to the Platform.
+- Impact user flows through tasks.
+
+For your product or feature, make a copy of this template and check it into products/platform/<product-name>.
+
```

Comment:
one more question on guidance: when do we think users can/should _close_ this ticket, and how can we walk them through those expectations?

I'd normally expect "launch" for a release plan template, but we also have a lot of "post-launch" information we ask people for in this template. 🤔 I'm open to ideas here

Reply:
I pulled the post-launch info from the VFS release plan template. They do post-launch reviews which I don't think we do (at least not today). Perhaps we can mark that section as Optional for users who want to document post-launch KPIs and say you can close the ticket after launch.

Reply:
I created a separate "Phase" for the post-launch items and indicated it's optional. Updated the Guidance instructions to say users can close the ticket at launch unless they want to document post-launch results.